lms/templates/admin/base.html.jinja2

```diff
@@ -12,8 +12,21 @@
                 {% endif %}
             {% endblock %}
         </title>
+        <link rel="stylesheet"
+              href="https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.15.2/css/selectize.default.min.css"
+              integrity="sha512-pTaEn+6gF1IeWv3W1+7X7eM60TFu/agjgoHmYhAfLEU8Phuf6JKiiE8YmsNC0aCgQv4192s4Vai8YZ6VNM6vyQ=="
```

Comment:
Selectize seemed like the less amount-of-work option here. It looks "decent" but a bit different that the rest of the components. IMO, good enough.